docs/using.rst

```diff
@@ -538,8 +538,14 @@ commands into your individual environment.
 Modifying the Renewal Configuration File
 ----------------------------------------
 
+When a certificate is issued Certbot creates a renewal configuration file that
+tracks the options that were selected when Certbot was run. This allows Certbot
+to use those same options again when it comes time for renewal. These renewal
+configuration files are located at ``/etc/letsencrypt/renewal/CERTNAME``.
+
 For advanced certificate management tasks, it is possible to manually modify the certificate's
-renewal configuration file, located at ``/etc/letsencrypt/renewal/CERTNAME``.
+renewal configuration file, but this is discouraged since it can easily break Certbot's
+ability to renew your certificates.
```

Comment:
It might be good to say here (or at least somewhere) that if you do manually modify this file, you should test it using `certbot renew --dry-run` to ensure it can be parsed correctly.